Source/Views/TableViews/SPCopyTable.m

```diff
@@ -1314,6 +1314,25 @@ - (BOOL)shouldUseFieldEditorForRow:(NSUInteger)rowIndex column:(NSUInteger)colIn
 	return NO;
 }
 
+/**
+ * Determine whether to use the sheet for editing; do so if the multipleLineEditingButton is enabled,
+ * or if the column was a blob or a text, or if it contains linebreaks.
+ */
+- (BOOL)shouldNotEditGeneratedFieldForColumn:(NSUInteger)colIndex
```

Comment:
Maybe we can tweak the name and flip it to positive - ex. isColunmnEditable maybe?

Reply:
Done....
I left the current alert message as it is the only control for now (Only check generated column). But in the future, if there are other checks for the editable column, the message should be reconsidered.